### ISSUE TITLE
Hangfire Dashboard - Npgsql.PostgresException (0x80004005): 42601: syntax error at or near "value"

### ISSUE THREAD

**Issue by pjninnis** [frankhommers/Hangfire.PostgreSql#287]
Hoping someone can help with the following...

I'm in the process of upgrading an old application from ASP.NET to ASP.NET Core,
and I'm getting the **Error below when calling the Hangfire Dashboard** URL (https://localhost:7169/hangfire).

This is for a completely fresh Hangfire database install (I was getting weird Postgres upgrade issues using the current database structure which I'll detail later on a separate ticket)

**It appears that Hangfire itself is working correctly as jobs are running correctly.**

**Error**
```
fail: Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware[1]
      An unhandled exception has occurred while executing the request.
      Npgsql.PostgresException (0x80004005): 42601: syntax error at or near "value"

      POSITION: 59
         at Npgsql.Internal.NpgsqlConnector.<ReadMessage>g__ReadMessageLong|225_0(NpgsqlConnector connector, Boolean async, DataRowLoadingMode dataRowLoadingMode, Boolean readingNotifications, Boolean isReadingPrependedMessage)
         at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming, CancellationToken cancellationToken)
         at Npgsql.NpgsqlDataReader.NextResult(Boolean async, Boolean isConsuming, CancellationToken cancellationToken)
         at Npgsql.NpgsqlDataReader.NextResult()
         at Dapper.SqlMapper.GridReader.NextResult() in /_/Dapper/SqlMapper.GridReader.cs:line 387
         at Dapper.SqlMapper.GridReader.ReadRow[T](Type type, Row row) in /_/Dapper/SqlMapper.GridReader.cs:line 196
         at Dapper.SqlMapper.GridReader.ReadSingle[T]() in /_/Dapper/SqlMapper.GridReader.cs:line 84
         at Hangfire.PostgreSql.PostgreSqlMonitoringApi.<GetStatistics>b__24_0(IDbConnection connection)
         at Hangfire.PostgreSql.PostgreSqlStorage.UseConnection[T](DbConnection dedicatedConnection, Func`2 func)
         at Hangfire.PostgreSql.PostgreSqlMonitoringApi.UseConnection[T](Func`2 func)
         at Hangfire.PostgreSql.PostgreSqlMonitoringApi.GetStatistics()
         at Hangfire.Dashboard.RazorPage.<Assign>b__44_0()
         at System.Lazy`1.ViaFactory(LazyThreadSafetyMode mode)
         at System.Lazy`1.ExecutionAndPublication(LazyHelper executionAndPublication, Boolean useDefaultConstructor)
         at System.Lazy`1.CreateValue()
         at System.Lazy`1.get_Value()
         at Hangfire.Dashboard.RazorPage.get_Statistics()
         at Hangfire.Dashboard.Pages.HomePage.Execute()
         at Hangfire.Dashboard.RazorPage.TransformText(String body)
         at Hangfire.Dashboard.RazorPage.ToString()
         at Hangfire.Dashboard.RazorPageDispatcher.Dispatch(DashboardContext context)
         at Hangfire.Dashboard.AspNetCoreDashboardMiddleware.Invoke(HttpContext httpContext)
         at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.InvokeCore(HttpContext context, PathString matchedPath, PathString remainingPath)
         at Microsoft.AspNetCore.SystemWebAdapters.PreBufferRequestStreamMiddleware.InvokeAsync(HttpContext context)
         at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
         at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
         at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
         at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
         at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
        Exception data:
          Severity: ERROR
          SqlState: 42601
          MessageText: syntax error at or near "value"
          Position: 59
          File: scan.l
          Line: 1149
          Routine: scanner_yyerror
```

### Environment
ASP.NET Core 7.0
.NET 6.0
PostgreSQL 12.6

### Nugets
Hangfire.AspNetCore 1.7.33
Hangfire.PostgreSql 1.9.10
Npgsql 7.0.2
Dapper 2.0.123

### Code (Startup)

```
services.AddHangfire(configuration => configuration
    .SetDataCompatibilityLevel(CompatibilityLevel.Version_170)
    .UseActivator(simpleInjector)
    .UseSimpleAssemblyNameTypeSerializer()
    .UseRecommendedSerializerSettings()
    .UsePostgreSqlStorage(
        databaseConnectionString,
        new PostgreSqlStorageOptions
        {
            InvisibilityTimeout = TimeSpan.FromHours(invisibilityTimeout),
            PrepareSchemaIfNecessary = true
        })
    .UseFilter(new PrioritiseAttribute())
    .UseFilter(new QueueNameFixer())
    .UseFilter(new QueueNameRecorder())
);

services.AddHangfireServer(options =>
{
    options.WorkerCount = workerCount;
    options.Queues = queues;
});

....

app.UseHangfireDashboard();
app.UseEndpoints(endpoints =>
{
    endpoints.MapControllers();
    endpoints.MapHangfireDashboard();
}

```

**Comment by azygis** [comment]
Fixed with #280. @frankhommers has to push the library to NuGet.

**Comment by tonykaralis** [comment]
@frankhommers could we get an ETA in the nuget release please :) @azygis thanks for the fix.
